x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -255,9 +257,23 @@
         },
         {
             "set": {
+                "if": "ctx.suricata?.eve?.event_type == \"alert\"",
+                "field": "event.kind",
+                "value": "alert"
+            }
+        },
+        {
+            "set": {
+                "if": "ctx.suricata?.eve?.event_type != \"alert\"",
```

Comment:
I think it would be useful to add a third kind.

Suricata optionally reports stats regularly. They're [very different beasts](https://github.com/elastic/beats/blob/7.0/x-pack/filebeat/module/suricata/eve/test/eve-small.log-expected.json#L192-L323) than the rest of the events. So perhaps we should use `event.kind:metrics` for them?

The category "network_traffic" doesn't apply to the stats events, either. So you'll have to adjust there as well.

Reply:
ok, I'll categorize the stats as `event.kind: metrics` and drop the `network_traffic`.